spec/requests/api/service_templates_spec.rb

```diff
@@ -171,4 +171,31 @@
       expect(response).to have_http_status(:no_content)
     end
   end
+
+  describe "service requests subcollection" do
+    it "can list a service template's service requests" do
```

Comment:
Hmm, we might need to add in the api.yml a service_requests_subcollection_actions section for service_templates for better control on who supports what (exposing different service_requests subcollection actions on service_templates vs. service_orders). We can review in next release. Thanks for noticing this.